Demo/Application/Features/ShopperInsightsViewController.swift

```diff
@@ -0,0 +1,150 @@
+    private let payPalCheckoutButton = UIButton(type: .system)
+    private let payPalVaultButton = UIButton(type: .system)
+    private let venmoButton = UIButton(type: .system)
+    
+    override init(authorization: String) {
+        let apiClient = BTAPIClient(authorization: authorization)!
+        
+        shopperInsightsClient = BTShopperInsightsClient(apiClient: apiClient)
+        paypalClient = BTPayPalClient(apiClient: apiClient)
+        venmoClient = BTVenmoClient(apiClient: apiClient)
+
+        super.init(authorization: authorization)
+        
+        title = "Shopper Insights"
+        
+        let shopperInsightsButton = UIButton(type: .system)
+        shopperInsightsButton.setTitle("Fetch recommended payments", for: .normal)
+        shopperInsightsButton.translatesAutoresizingMaskIntoConstraints = false
+        shopperInsightsButton.addTarget(self, action: #selector(shopperInsightsButtonTapped(_:)), for: .touchUpInside)
+        view.addSubview(shopperInsightsButton)
+        
+        payPalCheckoutButton.setTitle("PayPal Checkout", for: .normal)
+        payPalCheckoutButton.translatesAutoresizingMaskIntoConstraints = false
+        payPalCheckoutButton.addTarget(self, action: #selector(payPalCheckoutButtonTapped(_:)), for: .touchUpInside)
+        payPalCheckoutButton.isEnabled = false
+        view.addSubview(payPalCheckoutButton)
+        
+        payPalVaultButton.setTitle("PayPal Vault", for: .normal)
+        payPalVaultButton.translatesAutoresizingMaskIntoConstraints = false
+        payPalVaultButton.addTarget(self, action: #selector(payPalVaultButtonTapped(_:)), for: .touchUpInside)
+        payPalVaultButton.isEnabled = false
+        view.addSubview(payPalVaultButton)
+        
+        venmoButton.setTitle("Venmo", for: .normal)
+        venmoButton.translatesAutoresizingMaskIntoConstraints = false
+        venmoButton.addTarget(self, action: #selector(venmoButtonTapped(_:)), for: .touchUpInside)
+        venmoButton.isEnabled = false
+        view.addSubview(venmoButton)
+        
+        view.addConstraints([
+            shopperInsightsButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            shopperInsightsButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            payPalVaultButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            payPalVaultButton.bottomAnchor.constraint(equalTo: payPalCheckoutButton.bottomAnchor, constant: -40),
+            payPalCheckoutButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            payPalCheckoutButton.bottomAnchor.constraint(equalTo: venmoButton.bottomAnchor, constant: -40),
+            venmoButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            venmoButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -40)
+        ])
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
Again may be a punt but we can remove this entirely if we use `createPaymentButton` instead of the `init` override.

Reply:
Addressed in 92676b675c06beb21269bc7a19b94d9b8565525e